tests/test_ckan_to_frictionless.py

```diff
@@ -169,56 +169,82 @@ def test_unjsonify_all_extra_values(self):
         assert out == exp
 
     def test_dataset_license(self):
+        # No license_title nor license_url
```

Comment:
Mentioning because we're not storing `null` values in the Frictionless format (but we are from Frictionless to CKAN).